docs/root/configuration/http/http_filters/jwt_authn_filter.rst

```diff
@@ -130,6 +136,10 @@ JWT payload will be added to the request header as following format::
 
     x-jwt-payload: base64url_encoded(jwt_payload_in_JSON)
 
+JWT Claim ("sub" and "nested.claim.key") will be added to HTTP headers as following format::
+    x-jwt-claim-sub: <JWT Claim>
+    x-jwt-claim-nested-key: <JWT Claim>
+
```

Comment:
Needs an entry in https://github.com/envoyproxy/envoy/blob/main/changelogs/current.yaml

Reply:
sure, will add this.